includes/decoders/poc.py

```diff
@@ -123,16 +123,39 @@ def decode(freq, decoded):
 			logging.debug("POCSAG Bitrate: %s", bitrate)
 
 			if "Alpha:" in decoded: #check if there is a text message
-				poc_text = decoded.split('Alpha:   ')[1].strip().replace('<NUL><NUL>','').replace('<NUL>','').replace('<NUL','').replace('< NUL>','').replace('<EOT>','').strip()
-			else:
-				poc_text = ""
+               			poc_text = decoded.split('Alpha:   ')[1].strip().replace('<NUL><NUL>','').replace('<NUL>','').replace('<NUL','').replace('< NUL>','').replace('<EOT>','').strip()
+                		try:
+                    			logging.debug("Using %s to find geo-tag in %s", globalVars.config.get("POC","geo_format"),poc_text)
+                    			m = re.search(globalVars.config.get("POC","geo_format"),poc_text)

```

Comment:
Was passiert hier, wenn `geo_format`, also der RegEX Stiring NICHT gesetzt ist?
Kann hier eine Exception fliegen?

Reply:
Kann sie, wird allerdings abgefangen und es wird so gehandelt, als ob keine Auswertung stattgefunden hat; das data-Feld "has_geo" wird auf false gesetzt.

Reply:
Dann wäre es besser vor der Abfrage zu prüfen ob ein Regex vorhanden ist. Oder ein Enable Flag aus der Config lesen.
Eine Exception gewillt zu schmeißen ist nicht schön. Und nicht jeder wird diese Funktion nutzen wollen/können.

Reply:
Alles klar, ich setze dann noch ein Flag ein - perspektivisch kommt ja auch noch was mehr dazu (Geodecode mit Straße/PLZ), dann passt das

Reply:
Gibts hier schon Neuigkeiten?

Reply:
Ist in Arbeit

Reply:
Alles klar, super!
Wollte da jetzt auf keine Hektik rein bringen - war rein Interessehalber.
Danke dir.